tests/integration/component.go

```diff
@@ -340,6 +340,14 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", append(args, "delete", "--all", "-f")...)
 
 		})
+
+		It("creates a local nodejs component without pushing and then deletes it using --all flag", func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), context)
```

Comment:
```
$ odo catalog list components
NAME        PROJECT       TAGS
dotnet      openshift     2.0,latest
httpd       openshift     2.4,latest
nginx       openshift     1.10,1.12,1.8,latest
nodejs      openshift     10,6,8,8-RHOAR,latest
perl        openshift     5.24,5.26,latest
php         openshift     7.0,7.1,latest
python      openshift     2.7,3.5,3.6,latest
ruby        openshift     2.3,2.4,2.5,latest
wildfly     openshift     10.0,10.1,11.0,12.0,13.0,8.1,9.0,latest
```
We have many components in the catalog list. Can you please add some random component type like ```ruby, php, ...``` to add some variety to our test scenario. Moreover this also helps to verify the different default component shown in the catalog list.